### PR TITLE
fix: Removed require-ness of dtype

### PR DESCRIPTION
**Pull request by yuubd**

### CODE REVIEW

File: app/models/item_device.py [hunk 1]
```diff
@@ -35,10 +35,18 @@ def __str__(self):
...
+    :rtype: dict
+
+    for the device without its type, set it as MISCELLANEOUS
+    """
     def csv_dict(self):
         return {
-            "Category - Item Device Type": self.dtype.device_type,
-            "Type - Item Device Type": self.dtype.category,
+            "Category - Item Device Type": self.dtype.device_type
+                if self.dtype is not None else ITEM_MAP.get("")["device_type"],
```
Comment (bwdmonkey) [inline app/models/item_device.py:47]: Please make sure that the exported names of default `ITEM_MAP.get("")["device_type"]` matches that of the `ItemCategoryEnum` display name. Please see https://github.com/CodeTheChangeUBC/reBOOT/blob/master/app/models/item_device_type.py

File: app/views/views.py [hunk 2]
```diff
@@ -204,7 +204,7 @@ def _context(title, override={}):
 
 
 def _error(request: HttpRequest, err_msg="Something went wrong.", e=None):
-    # logger.exception(err_msg)
+    # logger.exception(e)
```
Comment (bwdmonkey) [inline app/views/views.py:207]: Feel free to delete the commented out logger

**Review by bwdmonkey — CHANGES_REQUESTED** [review]
I think the changes seem promising. See the minor comment I left and also see the requests changes by the CI. Thanks for the great work! Almost there 😄

File: app/views/views.py (hunk 2 above)
Comment (yuubd) [reply app/views/views.py:207]: I will leave it here ;) it was helpful.

**Comment by bwdmonkey** [comment]
@yuubd Just a FYI. https://www.conventionalcommits.org/en/v1.0.0-beta.2/ :) 

Also fixed a minor mapping issue cuz I thought it would take longer to explain than to solve haha. Maybe a quick at my changes. Well done. The changes works great :)

**Review by bwdmonkey — APPROVED** [review]

**Pull request merged**